src/core/dom/layout/index.js

```diff
@@ -61,6 +61,7 @@ export const LOADING_ELEMENTS_ENUM = {
   AMP_FACEBOOK_COMMENTS: 'AMP-FACEBOOK-COMMENTS',
   AMP_FACEBOOK_PAGE: 'AMP-FACEBOOK-PAGE',
   AMP_GOOGLE_DOCUMENT_EMBED: 'AMP-GOOGLE-DOCUMENT-EMBED',
+  AMP_GOOGLE_READ_ALOUD_PLAYER: 'AMP-GOOGLE-READ-ALOUD-PLAYER',
```

Comment:
Not necessary to list. Element tagNames ending with `-player` get this treatment by default.

This is important to remove since it increases the size of the main runtime.

```suggestion
```